serializations.md

```diff
@@ -744,6 +754,81 @@ NOTE: As the POI contains the Merkle Patricia Tree nodes (e.g. not only their ha
 * The key used for storing each object in each state subtree is not necessarily derived from the object itself.
 * The value(s) whose inclusion the POI proves is included in the POI itself.
 
+#### Merkle Patricia Value
+```
```

Comment:
According to the [implementation](https://github.com/aeternity/epoch/blob/master/apps/aeutils/src/aeu_mtrees.erl#L275-L281)